engine/src/main/java/org/terasology/rendering/world/WorldRendererImpl.java

```diff
@@ -247,25 +252,29 @@ private void initRenderGraph() {
         renderGraph.addNode(worldReflectionNode, "worldReflectionNode");
 
         // sky rendering
-        FBOConfig reflectedRefractedBufferConfig = new FBOConfig(RefractiveReflectiveBlocksNode.REFRACTIVE_REFLECTIVE_FBO, FULL_SCALE, FBO.Type.HDR).useNormalBuffer();
+        FBOConfig reflectedRefractedBufferConfig = new FBOConfig(RefractiveReflectiveBlocksNode.REFRACTIVE_REFLECTIVE_FBO_URI, FULL_SCALE, FBO.Type.HDR).useNormalBuffer();
         BufferClearingNode reflectedRefractedClearingNode = new BufferClearingNode(reflectedRefractedBufferConfig, displayResolutionDependentFBOs, GL_COLOR_BUFFER_BIT | GL_DEPTH_BUFFER_BIT);
         renderGraph.addNode(reflectedRefractedClearingNode, "reflectedRefractedClearingNode");
 
-        FBOConfig sceneOpaqueFboConfig = displayResolutionDependentFBOs.getFboConfig(READONLY_GBUFFER);
+        FBOConfig readOnlyGBufferConfig = displayResolutionDependentFBOs.getFboConfig(READONLY_GBUFFER);
+        FBOConfig writeOnlyGBufferConfig = displayResolutionDependentFBOs.getFboConfig(WRITEONLY_GBUFFER);
+
+        BufferClearingNode readOnlyGBufferClearingNode = new BufferClearingNode(readOnlyGBufferConfig, displayResolutionDependentFBOs, GL_COLOR_BUFFER_BIT | GL_DEPTH_BUFFER_BIT | GL_STENCIL_BUFFER_BIT);
+        renderGraph.addNode(readOnlyGBufferClearingNode, "readOnlyGBufferClearingNode");
 
-        BufferClearingNode readBufferClearingNode = new BufferClearingNode(sceneOpaqueFboConfig, displayResolutionDependentFBOs, GL_COLOR_BUFFER_BIT | GL_DEPTH_BUFFER_BIT | GL_STENCIL_BUFFER_BIT);
-        renderGraph.addNode(readBufferClearingNode, "readBufferClearingNode");
+        BufferClearingNode writeOnlyGBufferClearingNode = new BufferClearingNode(writeOnlyGBufferConfig, displayResolutionDependentFBOs, GL_COLOR_BUFFER_BIT | GL_DEPTH_BUFFER_BIT | GL_STENCIL_BUFFER_BIT);
```

Comment:
As I look at this I wonder if it'd be worth to create a GBufferClearingNode that also clear the additional buffers (i.e. normals and light accumulation) , perhaps taking advantage of an FBO.clear(mask) method that wraps things neatly. Something to think about, not to solve in this PR.